src/javascript/app_2/App/Components/Elements/FullPageModal/full_page_modal.jsx

```diff
@@ -0,0 +1,42 @@
+import React     from 'react';
+import PropTypes from 'prop-types';
+
+const FullPageModal = ({ title, body, onConfirm, confirmButtonText, onCancel, cancelButtonText, show }) => {
```

Comment:
Would be easier to compare with propTypes and more consistent if:
- Putting each property on a single line
- Alphabetical order like propTypes